.ci/release/Jenkinsfile

```diff
@@ -68,27 +69,30 @@ pipeline {
             }
           }
         }
-        //stage('Check master build status') {
-        //  steps {
-        //    script {
-        //      // If this build is not green: https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-mbp/job/master/
-        //      if(!buildStatus(host: 'apm-ci.elastic.co', job: ['apm-agent-java', 'apm-agent-java-mbp', 'master'], return_boolean: true)) {
-        //        input(message: "WARNING! The master build is not passing. Do you wish to continue?")
-        //      }
-        //    }
-        //  }
-        //}
+        stage('Check master build status') {
+        when { expression { params.ignore_branch_ci_status } }
+         steps {
+           script {
+             // If this build is not green: https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-mbp/job/master/
+             if(!buildStatus(host: 'apm-ci.elastic.co', job: ['apm-agent-java', 'apm-agent-java-mbp', 'master'], return_boolean: true)) {
+               input(message: "WARNING! The master build is not passing. Do you wish to continue?")
+             }
+           }
+         }
+        }
         stage('Require confirmation that CHANGELOG.asciidoc has been updated') {
           steps {
             input(message: """
             Update CHANGELOG.asciidoc to reflect the new version release:
             Go over PRs or git log and add bug fixes and features.
             Move release notes from the Unreleased sub-heading to the correct [[release-notes-{major}.x]] sub-heading (Example PR for 1.13.0 release).
 
-            Click 'Proceed' to confirm that this step has been completed or Abort to stop the build in order to complete this step.
+            Click 'Proceed' to confirm that this step has been completed and changes have been pushed or Abort to stop the build.
             """
             )
-            sh(script: "git pull")
+            dir("${BASE_DIR}") {
+              git credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba', url: 'git@github.com:elastic/apm-agent-java.git'
```

Comment:
I don't see any `deleteDir` step and the `BASE_DIR` should already contain the source code, What's the reason for cloning the repo again?

Reply:
Because we want to pull in any additional changes to the changelog that might be made as a result of the reminder and prompt just above this.

Reply:
In `branch_creation.sh`, we just do a plain git pull instead. Is that intentional?

Reply:
Good catch. I agree that we should be consistent here and I have replaced that shell call with the approach we are using in this step.